web/docs/library/examples/users.js

```diff
@@ -23,3 +23,55 @@ const user = await supabase.auth.user()
 export const logout = `
 await supabase.auth.logout()
 `.trim()
+
+export const requestRecoveryTokenEmail = `
+axios.post(
```

Comment:
I think we should just use `fetch` if we are going to give an example outside of our library. This documentation requires users to install a 3rd party library, without explaining how to install it. Also, we should probably use `async/await` syntax to be consistent with the rest of the docs.

Perhaps though, we should just ship this function now quickly? It shouldn't be hard right?